avocado/utils/vmimage.py

```diff
@@ -32,6 +32,13 @@
 from . import process
 
 
+#: The "qemu-img" binary used when creating the snapshot images.  If
+#: set to None (the default), it will attempt to find a suitable binary
+#: with :func:`avocado.utils.path.find_command`, which uses the the
+#: system's PATH environment variable
+QEMU_IMG = None
```

Comment:
Checking if I understood correctly... If I as an user of the vmimage module want to override qemu-img path then I do this:
```
import avocado.utils.vmimage
vmimage.QEMU_IMG = '/path/to/qemu-img'
```
If so then should it be documented because it may not be obvious to some users (like me:)?

Reply:
Yes, that's why it's commented with `#:`, it will be documented automatically with that long comment.

Reply:
Ok. Got it.